File: backend/src/main/java/com/smartcampus/backend/dto/weather/DailyWeatherDTO.java
package com.smartcampus.backend.dto.weather;

import lombok.*;

import java.time.LocalDate;

@Getter @Setter @NoArgsConstructor @AllArgsConstructor @Builder
public class DailyWeatherDTO {
    private LocalDate date;
    private Double temperatureMax;
    private Double temperatureMin;
    private Integer weathercode;
    private String description; // NOUVEAU - toujours variante "jour" (resume de la journee)
    private String icon;        // NOUVEAU
    private Double precipitationSum;
    private Double windspeedMax;
}

File: frontend/app/src/main/java/com/ObservatoireCampus/mobile/model/weather/DailyWeatherDto.kt
package com.ObservatoireCampus.mobile.model.weather

data class DailyWeatherDto(
    val date: String,
    val temperatureMax: Double?,
    val temperatureMin: Double?,
    val weathercode: Int?,
    val description: String?,
    val icon: String?,
    val precipitationSum: Double?,
    val windspeedMax: Double?
)